templates/eap-cd-image-stream.json

```diff
@@ -0,0 +1,87 @@
+{
+    "kind": "List",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "jboss-cd-image-streams",
+        "annotations": {
+            "description": "ImageStream definitions for JBoss Enterprise Application Platform Continuous Delivery.",
```

Comment:
I'm unclear about "Continuous Delivery" vs "continuous delivery" throughout this file.

Also, in eap71-image-stream.json it's "definition" not "definitions". I've no idea what's right; probably both are since 'items' is a single element array. :)

Reply:
As far as I'm aware these should all be lower case at this point, I'll update to be consistent.